selfdrive/thermald/fan_controller.py

```diff
@@ -83,20 +83,20 @@ def __init__(self) -> None:
     cloudlog.info("Setting up TICI fan handler")
 
     self.last_ignition = False
-    self.controller = PIController(k_p=0, k_i=2e-3, k_f=1, neg_limit=-80, pos_limit=0, rate=(1 / DT_TRML))
+    self.controller = PIController(k_p=2, k_i=2e-3, k_f=1, neg_limit=-80, pos_limit=0, rate=(1 / DT_TRML))
+    self.target = 75
 
   def update(self, max_cpu_temp: float, ignition: bool) -> int:
-    self.controller.neg_limit = -(80 if ignition else 30)
-    self.controller.pos_limit = -(30 if ignition else 0)
```

Comment:
Why was this removed in favor of the clip after the PID?

Reply:
Just feel cleaner to not change the limits of the PID and handle the ignition logic separately. but not needed.

Reply:
We do this in controls too. It ensures the integrators can't wind up past the limits. If you revert this change it should be good to merge.

Reply:
All set